apps/landing/components/layouts/Home/Navigation/index.tsx

```diff
@@ -1,42 +1,63 @@
 import type { FC } from 'react';
+import { StyleSheet } from 'react-native';
 import { useMedia } from '@tamagui/core';
-import { Button, Stack } from '@walless/ui';
+import { NavigationItem } from '@walless/app';
+import { View } from '@walless/gui';
+import { Button } from '@walless/ui';
 import { ContainerStack } from 'components/styled';
 import { handleShowSignUpModal } from 'features/home/HeadingSection/SignUpModal';
+import { useRouter } from 'next/router';
 
 import HomeButton from './HomeButton';
-import NavItem from './NavItem';
 import { navigationHeight, navigationItems } from './shared';
 
 export const HomeNavigation: FC = () => {
+	const router = useRouter();
 	const media = useMedia();
 
-	const temporarilyDisabled = true;
+	const temporarilyDisabled = false;
```

Comment:
will we use this `temporarilyDisabled` in the future? if not, may we remove them?